locust/main.py

```diff
@@ -423,6 +423,22 @@ def main():
         }
         console_logger.info(dumps(task_data))
         sys.exit(0)
+    
+    if options.run_time:
+        if not options.no_web:
+            logger.error("The --run-time argument can only be used together with --no-web")
```

Comment:
Why does it forbid this option if we are working from the web console ? 

In my use-case I would find it super useful to be able to **start the test from the web**, and have it stopping automatically even when I am disconnected / in a meeting **after some time** or when I hit the **stop button**. This would avoid me forgetting to stop the test and act as security to avoid a test running for ever...

Am I the only one who would like this feature not restricted to the ``--no-web`` ? Should I create an issue / feature request ?

Reply:
+1

Reply:
We would want this feature to be used with the web interface too, would be very useful.